src/styles/blocks/catalog.scss

```
@@ -14,12 +14,21 @@
 
     width: 200px;
     height: 300px;
-    border: 1px solid #ccc;
+    border: 1px solid $card-border-color;
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: color scale $transition-duration;
```

Your `transition` property doesn't work, so let's rewrite it

```suggestion
    transition: color $transition-duration, transform $transition-duration;
```